storage/Devices/BcacheImpl.cc

```diff
@@ -152,6 +217,15 @@ namespace storage
 	const File size_file = prober.get_system_info().getFile(SYSFS_DIR + get_sysfs_path() + "/size");
 
 	set_region(Region(0, size_file.get<unsigned long long>(), 512));
+
+	const File cache_mode_file = prober.get_system_info().getFile(SYSFS_DIR + get_sysfs_path() + "/bcache/cache_mode");
+        set_cache_mode(parse_mode(cache_mode_file));
+
+	const File writeback_percent_file = prober.get_system_info().getFile(SYSFS_DIR + get_sysfs_path() + "/bcache/writeback_percent");
+        set_writeback_percent(writeback_percent_file.get<int>());
```

Comment:
unsigned instead of int

Reply:
no, this won't work, as get is only specialized and has no specialization for unsigned ( I originally have it ). But as long as it is percents so 0..100 we are safe even with int.

Reply:
Well, since `File::get<int>()` contains "// TODO error checking", this is a full grown skeleton anyway.